### ISSUE TITLE
No type or protocol name 'FBSDKGraphRequestConnectionDelegate'

### ISSUE THREAD

**Issue by sadikay** [sbugert/react-native-admob#58]
My react native app working well. When i tried to import react-native-admob without CocoaPods to my app. I m getting _No type or protocol name 'FBSDKGraphRequestConnectionDelegate'_ error.

I tried to instal fbsdk again and I got this result:
```
rnpm-link info iOS module react-native-fbsdk is already linked

rnpm-install info Module react-native-fbsdk has been successfully installed & linked
```

I'm not sure if the problem on my fbsdk or react-native-admob.
Here is the part of my code when I got the error:

#import <RCTBridgeModule.h>
#import <Foundation/Foundation.h>

```
/*!
 @abstract A RCTFBSDKGraphRequestConnectionContainer object can take in a batch of
.........
 */

> @interface RCTFBSDKGraphRequestConnectionContainer : NSObject<FBSDKGraphRequestConnectionDelegate>

```

**Comment by sbugert** [comment]
So your app uses react-native-fbsdk and works but as soon as you add react-native-admob it stops working and shows this error?
This could indicate some parts of the teo libraries are not compatible. I have never used fbsdk before so I'm not sure.

Where is the error shown? In the app in the red background or in Xcode while compiling?

**Comment by sadikay** [comment]
Hi @sbugert yeah you got it correctly. The error shown while compiling in the RCTFBSDKGraphRequestConnectionContainer file. Actualy i m not using fbsdk too. It have been created after my  ```rnpm link``` command.

**Comment by sadikay** [comment]
I solved the issue by runing rnpm link only for admob:
 ```rnpm link react-native-admob```.

Then I got some errors while compiling for missing frameworks. I added them from
Build Phases -> Link Binary With Libraries
``` AddressBook.framework ``` 
``` libsqlite3.tbd ``` 

Then I added ```-lz``` to Build Settings -> Other Linker Flags. (See in the screenshoot)

<img width="750" alt="screen shot 2016-10-16 at 00 38 04" src="https://cloud.githubusercontent.com/assets/8470005/19413549/0a8e3d34-9339-11e6-8ac5-a14ca9fc04df.png">

Now my app works well! 👍 

_Note: Before adding react-native-admob my app was works well without this changes._
 
Thanks.
